issues-policy.md

```diff
@@ -0,0 +1,84 @@
+# GitHub issues process and policy
+
+The goals of the process are:
+
+1. Ensure errors or omissions in our docs are not blocking customer success.
+1. Be responsive to customer feedback and concerns.
+1. Continually improve the customer experience.
+1. Learn more about customer experiences through dialogue in the open on challenges and solutions.
+
+The process uses two stages to ensure responsiveness while prioritizing work. The initial stage diagnoses and triages the issue. The second stage resolves the issue. When an issue is both easy and urgent, the two stages may be combined.
+
+The process involves two time boxed tasks:
```

Comment:
time boxed --> time-boxed

Reply:
>time-boxed

I never heard the term before, it doesn't seem to be everyday language, can we omit it here?

Reply:
It's a common [scrum term](https://www.scruminc.com/what-is-timeboxing/) Should I add a link or define it here?

Reply:
Not everyone does scrum or is familiar with all of its terminology. If you want to keep it, making it a link should be good enough, as people will figure out what it means from context.

Reply:
I changed to 'time allocations' as a general term.